notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb

```diff
@@ -0,0 +1,1773 @@
+{
```

Comment:
<p>I see another "we" here. Replace throughout notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>